cmd/swarmctl/service/list.go

```diff
@@ -31,6 +31,17 @@ var (
 				return err
 			}
 
+			nr, err := c.ListNodes(common.Context(cmd), &api.ListNodesRequest{})
+			if err != nil {
+				return err
+			}
+			readyNodes := make(map[string]struct{})
+			for _, n := range nr.Nodes {
+				if n.Status.State == api.NodeStatus_READY {
```

Comment:
Agreed, but I think `swarmctl` should be consistent with `docker`. `docker` is checking for state == `READY`. Can you please file an issue or PR in docker/docker to change this?